app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class='container float-right'>
+  <div class='row align-items-center'>
+    <div class='col-5'>
+      <h2>My EMS Profile</h2>
+
+      <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+        <%= render "devise/shared/error_messages", resource: resource %>
+
+        <div class="field clearfix">
+          <%= f.label :full_name %>
+          <%= f.text_field :full_name, class: 'form-control'%>
+        </div>
+
+        <div class="field clearfix">
+          <%= f.label :email %>
+          <%= f.email_field :email, autofocus: true, autocomplete: 'email', class: 'form-control' %>
+        </div>
+
+        <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+          <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+        <% end %>
+
+        <div class="field" class="clearfix">
```

Comment:
Do we need `clearfix` class with 'field' class, we can remove it here and above if not required.